test/mitmproxy/console/test_flowlist.py

```diff
@@ -23,9 +14,9 @@ def mkmaster(self, **opts):
         o = options.Options(**opts)
         return console.master.ConsoleMaster(o, proxy.DummyServer())
 
-    @mock.patch('mitmproxy.tools.console.signals.status_message.send', side_effect = mock_log)
-    def test_new_request(self, test_func):
+    def test_new_request(self):
         m = self.mkmaster()
         x = flowlist.FlowListBox(m)
-        with pytest.raises(UrlError):
+        with Mock.patch('mitmproxy.tools.console.signals.status_message.send') as mock:
```

Comment:
Call it `mock_thing` instead of `mock`, then it doesn't collide with the import above.
This is how the official Python docs call it most of the time.